### DISCUSSION TITLE
文件路由是否需要处理大小写问题？

### DISCUSSION THREAD

**Discussion by 10Derozan** [web-infra-dev/rspress#518] [General]
目前，文件路由不会处理大小写，类如 Guide.mdx 生成的 routes.path 是 '/Guide'。
在 initPageData 时，
- https://github.com/web-infra-dev/rspress/blob/main/packages/core/src/runtime/App.tsx#L39 matchRoutes 方法对大小写不敏感
- https://github.com/web-infra-dev/rspress/blob/main/packages/core/src/runtime/App.tsx#L49 获取 pageInfo 时对大小写敏感

**Comment by sanyuan0704** [comment]
后者也可以做成大小写不敏感的
